Likhithanjali_practice

```diff
@@ -0,0 +1 @@
+
```

Comment:
****OccuranceDigitInNumber****
int count = 0;
        long temp = Math.abs(number); // Handle negative numbers

        // Count occurrences of the digit
        while (temp > 0) {
            if (temp % 10 == digit) {
                count++;
            }
            temp /= 10;
        }

Reply:
Create proper folder structure and make changes in that folder: 

eg `Likhithanjali-practice/OccuranceDigitInNumber.java`